src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -42,6 +42,15 @@
 short-summary: Manage contacts for certificate management.
 """
 
+helps['keyvault certificate contact add'] = """
+type: command
+short-summary: Add a contact to the specified vault to receive notifications of certificate operations.
+examples:
+  - name: Adds a certificate contact with the specified email to the Key Vault. (autogenerated)
```

Comment:
```suggestion
  - name: Add a certificate contact with the specified email to the Key Vault. (autogenerated)
```
To align with other examples